src/test/java/com/feelmycode/parabole/controller/EventControllerTest.java

```diff
@@ -145,24 +154,38 @@ public void createEvent() throws JSONException {
                         .host("parabole.com"),
                     prettyPrint()),
                 preprocessResponse(prettyPrint()),
+                requestFields(
+                    fieldWithPath("userId").type(JsonFieldType.NUMBER).description("사용자 아이디"),
+                    fieldWithPath("createdBy").type(JsonFieldType.STRING).description("이벤트 생성자"),
+                    fieldWithPath("type").type(JsonFieldType.STRING).description("이벤트 유형"),
+                    fieldWithPath("title").type(JsonFieldType.STRING).description("이벤트 제목"),
+                    fieldWithPath("startAt").type(JsonFieldType.STRING).description("이벤트 시작 일시"),
+                    fieldWithPath("endAt").type(JsonFieldType.STRING).description("이벤트 종료 일시"),
+                    fieldWithPath("descript").type(JsonFieldType.STRING).description("이벤트 설명"),
+                    fieldWithPath("eventImage").type(JsonFieldType.OBJECT).description("이벤트 이미지"),
+                    fieldWithPath("eventImage.eventBannerImg").type(JsonFieldType.STRING)
+                        .description("이벤트 배너 이미지"),
+                    fieldWithPath("eventImage.eventDetailImg").type(JsonFieldType.STRING)
+                        .description("이벤트 상세 이미지"),
+                    fieldWithPath("eventPrizeCreateRequestDtos").type(JsonFieldType.ARRAY)
+                        .description("이벤트 경품 정보"),
+                    fieldWithPath("eventPrizeCreateRequestDtos.[].id").type(JsonFieldType.NUMBER)
+                        .description("이벤트 경품 ID"),
+                    fieldWithPath("eventPrizeCreateRequestDtos.[].type").type(JsonFieldType.STRING)
+                        .description("이벤트 경품 유형"),
```

Comment:
유형은 어떤게 있는지 어떤식으로 전송하는지 설명해주세요

아래는 예시고 알맞게 수정해주세요

```suggestion
                    fieldWithPath("eventPrizeCreateRequestDtos.[].type").type(JsonFieldType.STRING)
                        .description("이벤트 경품 유형 (상품: PRODUCT /쿠폰: COUPON )"),
```